piksi_tools/console/update_downloader.py

```diff
@@ -62,24 +57,18 @@ def download_multi_firmware(self, hwrev):
         except KeyError:
             raise KeyError(
                 "Error downloading firmware: URL not present in index")
-        except URLError:
-            raise URLError(
-                "Error: Failed to download latest Multi firmware from Swift Navigation's website"
-            )
         return filepath
 
     def _download_file_from_url(self, url):
         if not os.path.isdir(self.root_dir):
```

Comment:
I changed this exception if the directory doesn't exist; this is a known error case that we try and inform the user about.  IOError is often something we don't predict.